test/package.js

```diff
@@ -21,3 +21,21 @@ test('Every rule is defined in index file', async t => {
 	t.is(Object.keys(index.configs.recommended.rules).length, ruleFiles.length,
 		'There are more exported rules in the recommended config than rule files.');
 });
+
+test('Every fixable rule got valid meta.type', async t => {
+	const validTypes = ['problem', 'suggestion', 'layout'];
+
+	const files = await pify(fs.readdir)('rules');
+	const ruleFiles = files.filter(file => path.extname(file) === '.js');
```

Comment:
This is duplicated from the other rule and could just be moved to the top-level of the file and reused.